internal/policygen/iam.go

```diff
@@ -85,3 +128,18 @@ func validate(instance map[string]interface{}, mandatoryFields []string) error {
 	}
 	return nil
 }
+
+func projectNumber(rn runner.Runner, id string) (string, error) {
+	cmd := exec.Command("gcloud", "projects", "describe", id, "--format=value(projectNumber)")
```

Comment:
This is fine as well, but my preference is for JSON because that will be consistent with any gcloud lookup (e.g. if you want to get a list of values, etc).

Reply:
Have we documented gcloud as a dep?

Reply:
Added to README